pkg/collector/corechecks/snmp/internal/checkconfig/config_metric_test.go

```diff
@@ -43,3 +43,54 @@ func Test_normalizeRegexReplaceValue(t *testing.T) {
 		})
 	}
 }
+
+func Test_getMappedValue(t *testing.T) {
+	tests := []struct {
+		val                 string
+		mapping             map[string]string
+		expectedMappedValue string
+		expectedError       string
+	}{
+		{
+			"1",
+			map[string]string{
+				"1": "one",
+			},
+			"one",
+			"",
```

Comment:
I think it helps readability by naming each field of the struct like this:

```suggestion
			val: "1",
			mapping: map[string]string{
				"1": "one",
			},
			expectedMappedValue: "one",
```

It also avoids having the declare `expectedError` when the value if empty.